tests/integration/run_jsonrpc_commands.py

```diff
@@ -8,6 +8,14 @@
 import getopt
 import jsondiff
 
+def get_silkrpc_target(silk: bool, method: str):
```

Comment:
Probably a better name could be `get_target` because the condition on `silk` boolean is to differentiate between Silkrpc and RpcDaemon, so the target is not always Silkrpc :smile: